MDANSE/Src/MDANSE/Chemistry/ChemicalSystem.py

```diff
@@ -155,21 +173,21 @@ def get_substructure_matches(
 
```

Comment:
Since you're cleaning up here, the function above could be:
```python
        matches = self.rdkit_mol.GetSubstructMatches(
            Chem.MolFromSmarts(smarts), maxMatches=maxmatches
        )
        return {ind for match in matches for ind in match}
```